app/src/main/java/com/example/tung/moviedb_07/screen/list_item/ListItemViewModel.java

```diff
@@ -1,8 +1,12 @@
 package com.example.tung.moviedb_07.screen.list_item;
 
+import com.example.tung.moviedb_07.data.model.Genre;
+import com.example.tung.moviedb_07.data.model.Movie;
 import com.example.tung.moviedb_07.screen.BaseRecyclerViewAdapter;
 import com.example.tung.moviedb_07.screen.BaseViewModel;
+import com.example.tung.moviedb_07.utils.Constant;
 import com.example.tung.moviedb_07.utils.navigator.Navigator;
+import java.util.List;
```

Comment:
ERROR: Unused import: java.util.List. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports